mlrun/datastore/store_resources.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2018 Iguazio
```

Comment:
File should be renamed to `resources_store.py` no ?

Reply:
no, as i wrote its access to the `store://` resources , and not a store of resources